scripts/!mods_preload/mod_reforged.nut

```diff
@@ -133,3 +133,49 @@ foreach (requirement in requiredMods)
 	}
 	delete ::Reforged.QueueBucket;
 }, ::Hooks.QueueBucket.FirstWorldInit);
+
+// Helper function to move an already registered unit to the end of the vanilla tables, so that savegames remain compatible
```

Comment:
This should be reverted before merging into the main build. We could make a save-port patch if possible separately.